src/parser/serializer.ts

```diff
@@ -676,7 +690,7 @@ export function serializeInto(
       } else if (typeof value === 'number') {
         index = serializeNumber(buffer, key, value, index);
       } else if (typeof value === 'bigint') {
-        throw new BSONTypeError('Unsupported type BigInt, please use Decimal128');
+        index = serializeBigInt(buffer, key, value, index);
```

Comment:
If you run `npm run check:coverage` you can actually view what lines of code get hit by our testing, and we currently don't hit this or the line in the Map section either, can we add two tests covering each code path?